src/Components/Wizard/steps/Pubkeys/NewKeyForm.js

```diff
@@ -95,12 +118,14 @@ const NewSSHKeyForm = ({ setStepValidated }) => {
         <FileUpload
           id="public-key-value"
           onDataChange={handleSSHKeyChange}
-          allowEditingUploadedText={false}
+          allowEditingUploadedText={true}
+          onTextChange={handleSSHKeyText}
```

Comment:
can't we use the `handleSSHKeyChange`? looks similar to me?

Reply:
they differ, we do not want to upload invalid key, but we want to allow adding more text to a field that is not valid yet :)

Reply:
it might be similar and possible to join these methods, but I personally would introduce more bugs then if I write two xD